UI/BuildingsPanel.cpp

```diff
@@ -107,8 +107,13 @@ void BuildingsPanel::Update() {
         // skip known destroyed and stale info objects
         if (this_client_known_destroyed_objects.contains(object_id))
             continue;
-        if (this_client_stale_object_info.contains(object_id))
-            continue;
+        if (this_client_stale_object_info.contains(object_id)) {
+            if (GetOptionsDB().Get<bool>("ui.map.scanlines.shown") && planet->Owner() == this_client_empire_id)
```

Comment:
is this the right option? not `ui.map.sidepanel.stale-buildings.shown` ?

Reply:
I suppose there's no need to have the planet owner check either... if a player has stale visibility buildings on another player's planet, why not allow showing and dismissing them?